docs/sources/datasources/tempo/query-editor/_index.md

```diff
@@ -26,86 +26,167 @@ refs:
+![The Search query builder](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-search-v11.png)
 
-### TraceQL query editor
+### TraceQL Query editor
 
-The TraceQL query editor, located on the **Explore** > **TraceQL** tab in Grafana, lets you search by trace ID and write TraceQL queries using autocomplete.
+The TraceQL query editor lets you search by trace ID and write TraceQL queries using autocomplete.
 
 Refer to the [TraceQL query editor documentation]({{< relref "./traceql-editor" >}}) to learn more about constructing queries using a code-editor-like experience.
 
-![The TraceQL query editor](/static/img/docs/tempo/screenshot-traceql-query-editor-v10.png)
+![The TraceQL query editor](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-traceql-v11.png)
+
+You can also search for a Trace ID by entering a trace ID into the query field.
+
+### Service graph view
+
+Grafana’s service graph view uses metrics to display span request rates, error rates, and durations, as well as service graphs.
+Once the requirements are set up, this pre-configured view is immediately available.
+
+Using the service graph view, you can:
+
+- Discover spans which are consistently erroring and the rates at which they occur
+- Get an overview of the overall rate of span calls throughout your services
+- Determine how long the slowest queries in your service take to complete
+- Examine all traces that contain spans of particular interest based on rate, error, and duration values (RED signals)
+
+For more information about the service graph, refer to [Service graph](../service-graph/).
+
+![Screenshot of the Service Graph view](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png)
+
+## Use TraceQL panels in dashboards
+
+To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
+
+To learn more about Grafana dashboards, refer to the [Use dashboards documentation](/docs/grafana/latest/dashboards/use-dashboards/).
+
+## Set options for query builder and editor
+
+The following options are available for the **Search** and **TraceQL** query types.
+You can modify these settings in the **Options** section.
+
+![Options section in the query editors](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-options.png)
+
+After changing any option, re-run the query to apply the updates.
+
+Limit
+: Determines the maximum number of traces to return. Default value is `20`.
+
+Span Limit
+: Sets the maximum number of spans to return for each spanset. Default value is `3`.
+
+Table Format
+: Determines whether the query results table is displayed focused on **Traces** or **Spans**. **Traces** is the default selection. When **Traces** is selected, the results table starts with the Trace ID. When **Spans** is selected, the table starts with the Trace Service.
```

Comment:
lowercasing these unless they're UI elements that you're calling out, in which case they should be bold
```suggestion
: Determines whether the query results table is displayed focused on **Traces** or **Spans**. **Traces** is the default selection. When **Traces** is selected, the results table starts with the trace ID. When **Spans** is selected, the table starts with the trace dervice.
```